stock/templates/registered/fleetup.html

```diff
@@ -0,0 +1,221 @@
+{% extends "public/base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+
+{% block title %}Alliance Auth{% endblock %}
+{% block page_title %}FleetUp{% endblock page_title %}
+
+{% block content %}
+<div class="col-lg-12">
+<nav class="navbar navbar-default">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+            <a class="navbar-brand" href="#">Fleet-Up</a>
+        </div>
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul class="nav navbar-nav">
+                <li class="active"><a href="#">Ops and Timers <span class="sr-only">(current)</span></a></li>
+                <li><a href="/fleetup/doctrines/">Doctrines</a></li>
+                <li><a href="/fleetup/fittings/">Fittings</a></li>
+                {% if perms.auth.corp_stats %}
```

Comment:
the corp_stats permission was removed from the corp stats fyi it only uses two permissions now alliance_api's and corp_api's